kratos/containers/variables_list_data_value_container.h

```diff
@@ -333,7 +333,6 @@ class KRATOS_API(KRATOS_CORE) VariablesListDataValueContainer
     TDataType& GetValue(const Variable<TDataType>& rThisVariable, SizeType QueueIndex)
     {
         KRATOS_ERROR_IF_NOT(mpVariablesList->Has(rThisVariable)) << "This container only can store the variables specified in its variables list. The variables list doesn't have this variable:" << rThisVariable << std::endl;
-        KRATOS_DEBUG_ERROR_IF((QueueIndex + 1) > mQueueSize) << "Trying to access data from step " << QueueIndex << " but only " << mQueueSize << " steps are stored." << std::endl;
         return *(TDataType*)Position(rThisVariable, QueueIndex);
```

Comment:
Here we woud check for the variable name twice (once in GetValue, once in Position)

Reply:
I don't think the method being slower in debug is really a problem, I was just commenting because I believe that the check in line 335 is the same as that in 971, which would be called from here.

Reply:
There is a difference since like 335 is checking with or without debug. This seems to be the only difference between GetValue and FastGetValue.

Reply:
Oh you are right... I did not realize this macro is not debug-only. Disregard the comment then.